charts/jenkins/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
```

Comment:
Obsolete, this is used by the upstream (https://github.com/kubernetes/charts) to define some permissions.

Reply:
This is added by running: `helm fetch --untar stable/jenkins ` in charts directory. Do we need to clean the upstream every time? I think the pro of Helm is to get quickly a community charts and install them to your cluster. As they have no impact, cleaning them may be over kill, I think, no?

Reply:
Valid point, reason why it works for the `redis` chart in that case is that it's included in their `.helmignore`. May be worth pushing a fix to the upstream.

Reply:
I made a PR on the Jenkins chart. https://github.com/kubernetes/charts/pull/5821